### PR TITLE
PD-236 filtering output with jq

### PR DESCRIPTION
**Pull request by nathan-px**
@sangleganesh can you help me write this one up? I made some assumptions behind the reasoning for why jq is used, and I couldn't get all of the sample commands you listed in JIRA to work on my cluster.

### CODE REVIEW

**Comment by nathan-px** [comment]
Reviewed with @sangleganesh, these steps should now be technically accurate.

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/filtering-output-with-jq.md [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+description: Find out how to filter pxctl output using jq
+---
+
+The pxctl command line output displays a limited amount of information based on the context of the subcommand.
+
+You can view all available output information using the `-j` option, which supplies the full output in JSON. However, this output is verbose, making it difficult to find relevant information manually. To improve the output, you can further parse the JSON inline using the [jq command-line processor](https://stedolan.github.io/jq/).
+
+Using jq, you can filter the output to display only the information want which might not be available on the standard command line output, or would be difficult to find in the full JSON output.
+
+This document provides instructions for filtering pxctl's JSON output using jq for the following common use cases:
```
Comment (sangleganesh) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/filtering-output-with-jq.md]: s/instructions/examples

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/filtering-output-with-jq.md [hunk 2]
```diff
@@ -0,0 +1,399 @@
...
+title: Filtering output with jq
+keywords: portworx, container, Kubernetes, storage, Docker, k8s, pv, persistent disk, monitoring, prometheus, alertmanager, servicemonitor, grafana, lighthouse, px-central, px-kvdb, jq
+description: Find out how to filter pxctl output using jq
+---
+
+The pxctl command line output displays a limited amount of information based on the context of the subcommand.
+
+You can view all available output information using the `-j` option, which supplies the full output in JSON. However, this output is verbose, making it difficult to find relevant information manually. To improve the output, you can further parse the JSON inline using the [jq command-line processor](https://stedolan.github.io/jq/).
+
+Using jq, you can filter the output to display only the information want which might not be available on the standard command line output, or would be difficult to find in the full JSON output.
```
Comment (sangleganesh) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/filtering-output-with-jq.md]: information want -> information you want ?

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/filtering-output-with-jq.md [hunk 3]
```diff
@@ -0,0 +1,399 @@
+---
+title: Filtering output with jq
+keywords: portworx, container, Kubernetes, storage, Docker, k8s, pv, persistent disk, monitoring, prometheus, alertmanager, servicemonitor, grafana, lighthouse, px-central, px-kvdb, jq
+description: Find out how to filter pxctl output using jq
+---
+
+The pxctl command line output displays a limited amount of information based on the context of the subcommand.
+
+You can view all available output information using the `-j` option, which supplies the full output in JSON. However, this output is verbose, making it difficult to find relevant information manually. To improve the output, you can further parse the JSON inline using the [jq command-line processor](https://stedolan.github.io/jq/).
```
Comment (sangleganesh) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/filtering-output-with-jq.md]: I would remove 'However, this output is verbose, making it difficult to find relevant information manually.' .

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/filtering-output-with-jq.md [hunk 4]
```diff
@@ -0,0 +1,399 @@
...
+Retrieve your node names by entering the `pxctl cluster provision-status` command:
+```text
+pxctl cluster provision-status
+```
+<!-- there's also pxctl status, but this works too -->
+
+The cluster in this example comprises 3 worker nodes:
+
+```output
+NODE                                    NODE STATUS     POOL    POOL STATUS     IO_PRIORITY     SIZE    AVAILABLE      USED    PROVISIONED     RESERVEFACTOR   ZONE    REGION  RACK
```
Comment (sangleganesh) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/filtering-output-with-jq.md]: will need too update this as well ? (the examples below don't matchup). Here are the details:

```
/opt/pwx/bin/pxctl cluster provision-status
NODE					NODE STATUS	POOL	POOL STATUS	IO_PRIORITY	SIZE	AVAILABLE	USED	PROVISIONED	RESERVEFACTOR	ZONE	REGION	RACK
598b4c37-459f-45f6-9fbe-14ea0fdd31df	Up		0	Online		LOW		30 GiB	26 GiB		3.5 GiB	7.0 GiB		0		default	default	default
9a58c096-5085-4e9f-8094-8f341ebaab7a	Up		0	Online		LOW		30 GiB	26 GiB		3.5 GiB	1.0 GiB		0		default	default	default
```

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/filtering-output-with-jq.md [hunk 5]
```diff
@@ -0,0 +1,399 @@
...
+        "RuntimeState": "clean"
+      }
+    }
+  ],
+  "error": "",
+  "fs_resize_required": false
+}
+```
+
+## List volumes attached on a node
```
Comment (sangleganesh) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/filtering-output-with-jq.md]: List volumes with HA-Level 1

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/filtering-output-with-jq.md [hunk 6]
```diff
@@ -0,0 +1,399 @@
...
+
+You can see information about volumes attached to a given node. Consider the following hypothetical scenario: you have a new policy that no users should run unreplicated volumes. Since this is a new policy, your users have been creating volumes that have a mixture of ha-levels: some of only 1, some of 2. You don't know what volumes these are, and you need to make sure there are no volumes that are none with an ha-level of 1. You want to show the volume ID for any volume with an ha-level of 1.
```
Comment (sangleganesh) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/filtering-output-with-jq.md]: remove "You can see information about volumes attached to a given node. "

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/filtering-output-with-jq.md [hunk 7]
```diff
@@ -0,0 +1,399 @@
...
+
+Enter the following `pxctl volume list` command, specifying your own node ID, to list all volumes attached to that node:
```
Comment (sangleganesh) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/filtering-output-with-jq.md]: update this as well

**Pull request merged**